esp/esp/program/modules/handlers/studentregphasezero.py

```diff
@@ -59,7 +59,7 @@ def students(self, QObject = False):
         return {'phasezero': ESPUser.objects.filter(q_phasezero).distinct()}
 
     def studentDesc(self):
-        return {'phasezero': """Students who have entered the Student Lottery."""}
+        return {'phasezero': """Students who have entered the Student Lottery"""}
```

Comment:
I guess this isn't a sentence, so there shouldn't be a period, but if you're gonna take it out here, you better take it out of all of the other list descriptions!

Reply:
fair. done.